detection_rules/packaging.py

```diff
@@ -131,7 +131,7 @@ def _package_notice_file(save_dir):
             notice_txt = f.read()
 
         with open(os.path.join(save_dir, 'notice.ts'), 'wt') as f:
-            commented_notice = [' * ' + line for line in notice_txt.splitlines()]
+            commented_notice = [(' * ' + line).rstrip() for line in notice_txt.splitlines()]
```

Comment:
nit
```suggestion
            commented_notice = [f' * {line'`.rstrip() for line in notice_txt.splitlines()]
```

Reply:
not sure how the linted version is more clear? also has a stray \`

Reply:
ah i see. okay